src/layout.ts

```diff
@@ -589,6 +589,63 @@ export const splitAroundWhitespace = (s: string): string[] =>
     .split(SPLIT_MARKER)
     .filter((s) => s !== "");
 
+export const stringifyStyle = (style: PIXI.TextStyle): string => {
+  const {
+    align,
+    breakWords,
+    dropShadow,
+    dropShadowAlpha,
+    dropShadowAngle,
+    dropShadowBlur,
+    dropShadowColor,
+    dropShadowDistance,
+    fill,
+    fillGradientType,
+    fillGradientStops,
+    letterSpacing,
+    lineHeight,
+    lineJoin,
+    miterLimit,
+    padding,
+    stroke,
+    strokeThickness,
+    textBaseline,
+    trim,
+    whiteSpace,
+    wordWrap,
+    wordWrapWidth,
+    leading,
+  } = style;
+
+  return JSON.stringify({
+    align,
```

Comment:
Maybe you can do the same thing with object destructuring? e.g. `JSON.stringify(...style, { /* properties you want to omit or override */ });`